monai/networks/nets/densenet.py

```diff
@@ -148,6 +148,9 @@ class DenseNet(nn.Module):
     """
     Densenet based on: `Densely Connected Convolutional Networks <https://arxiv.org/pdf/1608.06993.pdf>`_.
     Adapted from PyTorch Hub 2D version: https://pytorch.org/vision/stable/models.html#id16.
+    This network is non-determistic When `spatial_dims` is 3 and CUDA is enabled. Please check the link below
```

Comment:
I think it'll only affect the training (with backprop), the forward part is deterministic could you help double check? Thanks!

Reply:
Thanks @wyli , yes, if only do forward part, it is not affected. Should it be mentioned in the doc-strings?

Reply:
thank you, I think the docstring is fine...